Handler error in latest version

### 🔍 Before submitting the issue

- [X] I have searched among the existing issues
- [X] I am using a Python virtual environment

### 🐞 Description of the bug

When running the docs I'm getting this error:

```output
writing output... [ 87%] mapdl_commands/solution/_autosummary/ansys.mapdl.core.Mapdl.bfv .. mapdl_commands/solution/_autosummary/ansys.mapdl.core.Mapdl.mfelem
Sphinx parallel build error:
sphinx.errors.ExtensionError: Handler <function update_and_remove_templates at 0x7f55445a41f0> for event 'html-page-context' threw an exception (exception: 'ansys' is not in list)
make: *** [Makefile:21: html] Error 2
```
https://github.com/ansys/pymapdl/actions/runs/5870157592/job/15917918928?pr=2248#step:15:231



### 📝 Steps to reproduce

It happens in PyMAPDL CICD.

### 💻 Which operating system are you using?

Linux

### 📀 Which ANSYS version are you using?

N/A

### 🐍 Which Python version are you using?

3.9

### 📦 Installed packages

```shell
NA
```

--- a/src/ansys_sphinx_theme/__init__.py
+++ b/src/ansys_sphinx_theme/__init__.py
@@ -137,7 +137,7 @@ def fix_edit_html_page_context(
     see https://github.com/pyvista/pyvista/pull/4113
     """
 
-    def fix_edit_link_button(link: str) -> str:
+    def fix_edit_link_page(link: str) -> str:
         """Transform "edit on GitHub" links to the correct URL.
 
         This function fixes the URL for the "edit this page" link.
@@ -203,7 +203,7 @@ def fix_edit_html_page_context(
         else:
             return link
 
-    context["fix_edit_link_button"] = fix_edit_link_button
+    context["fix_edit_link_page"] = fix_edit_link_page
 
 
 def update_footer_theme(

--- a/src/ansys_sphinx_theme/theme/ansys_sphinx_theme/components/edit-this-page.html
+++ b/src/ansys_sphinx_theme/theme/ansys_sphinx_theme/components/edit-this-page.html
@@ -1,7 +1,7 @@
 {% if sourcename is defined and theme_use_edit_page_button==true and
 page_source_suffix %} {% set src = sourcename.split('.') %}
 <div class="tocsection editthispage">
-  <a href="{{ fix_edit_link_button(get_edit_provider_and_url()[1]) }}">
+  <a href="{{ fix_edit_link_page(get_edit_provider_and_url()[1]) }}">
     <i class="fa-solid fa-pencil"></i>
     {% set provider = get_edit_provider_and_url()[0] %} {% block
     edit_this_page_text %} {% if provider %} {% trans provider=provider %}Edit

